OKTA-530036 Event Hook Guide image update (#3848)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/index.md

To complete the one-time verification of the event hook:

* After creating the event hook, and:

  * Your local application server is running (`>node server.js`).

>   * Your local ngrok session is running (and the forwarding URL is the same as the URL field defined in the Event hook)

Reviewer: ```suggestion
  * Your local ngrok session is running (and the forwarding URL is the same as the URL field defined in the Event hook).
```

1. Ensure that both your ngrok session and local sample application are running.
1. In your Okta org, sign in as an administrator and create a test user in the Admin Console.
    * Go to **Directory** > **People**, and click **Add person**. As an example, add the user John Doe with the following fields:
        * **First name:** John
        * **Last name:** Doe
        * **Username:** john.doe@example.com

1. Navigate back to your local application's console. If the event hook was successful, the following message appears:

    `The user john.doe@example.com has been added to the Okta org!`

> 1. Check your ngrok inspection interface (`http://localhost:4040`). Each call recorded by ngrok appears in the interface, from which you can review the complete call response body, header, and other details. See [Review ngrok inspection interface](/docs/guides/event-hook-ngrok/nodejs/main/#review-ngrok-inspection-interface). Clicking **Replay** on the ngrok inspection interface replays the event hook call to assist in development. See [Replay the event hook](#replay-the-event-hook).

Reviewer: ```suggestion
1. Check your ngrok inspection interface (`http://localhost:4040`). Each call recorded by ngrok appears in the interface from which you can review the complete call response body, header, and other details. See [Review ngrok inspection interface](/docs/guides/event-hook-ngrok/nodejs/main/#review-ngrok-inspection-interface). Click **Replay** on the ngrok inspection interface to replay the event hook call to assist in development. See [Replay the event hook](#replay-the-event-hook).
```

     `The user john.doe@example.com has been added to the Okta org!`

> 1. Check your ngrok inspection interface (`http://localhost:4040`). Each call recorded by ngrok appears in the interface, from which you can review the response body, header, and other details. See [Review ngrok inspection interface](/docs/guides/event-hook-ngrok/nodejs/main/#review-ngrok-inspection-interface).

Reviewer: ```suggestion
1. Check your ngrok inspection interface (`http://localhost:4040`). Each call recorded by ngrok appears in the interface from which you can review the response body, header, and other details. See [Review ngrok inspection interface](/docs/guides/event-hook-ngrok/nodejs/main/#review-ngrok-inspection-interface).
```
